src/net.h

```diff
@@ -662,6 +673,7 @@ struct CNodeOptions
     bool prefer_evict = false;
     size_t recv_flood_size{DEFAULT_MAXRECEIVEBUFFER * 1000};
     bool use_v2transport = false;
+    bool count_failure_after_reconnect = false;
```

Comment:
6a2dbed1209b18d923da64b019f6f378d3c30695: The word `count` is ambigous here, because `AddrManImpl` literally counts failures, but I think the meaning here is "it counts / we care about it".

See also my above comment. Depending how this is intended to work, maybe I can think of a better name.